lib/_emerge/depgraph.py

```diff
@@ -4616,6 +4633,12 @@ def _priority(self, **kwargs):
             priority_constructor = DepPriority
         return priority_constructor(**kwargs)
 
+    def _cross(self, eroot):
+        """
+        Returns True if the ROOT for  the given EROOT is not /.
```

Comment:
```suggestion
        Returns True if the ROOT for the given EROOT is not /.
```

Reply:
I've adjusted this method and docstring to also account for the cross-prefix case.

Reply:
thanks!